service/src/main/java/uk/gov/hmcts/reform/fpl/validators/DateOfBirthValidator.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.fpl.validators;
```

Comment:
unit tests for this? tests for before date, after date, on the same day etc would be a good start